### PR TITLE
Redux in components

### PR DESCRIPTION
**Pull request by felixDev22**
**In this repo**

- [ ] Changed the default state in the books reducer from an empty array to an array with a few books.
- [ ] Imported Redux `Provider` and store in the main component of React app.
- [ ] Displayed books from Redux store.
- [ ] Used `useSelector` to consume the state.
- [ ] Imported Redux `useDispatch`.
- [ ] Generated a unique id and added them to an object
- [ ] Added a functionality for "Remove" on button click
- [ ] Added a functionality for "Check status" button click
- [ ] Checked that it's working properly 😄

### CODE REVIEW

File: src/components/pages/Categories.js [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+    const notice = { text: 'Under construction' };
+    dispatch(checkStatus(notice));
+  };
+  const statusCheck = useSelector((state) => state.Categories);
+
+  return (
+    <div className="">
+      <button type="button" className="check-status" onClick={handleClick}>
+        check status
+      </button>
```
Comment (rloterh) [inline src/components/pages/Categories.js:17]: - [ ] Clicking on the Categories nav link navigates to a blank page, with the attached error thrown in the console, as can be seen below. Please ensure that your check status button is properly implemented and should display the message 'Under construction when clicked.

**Categories Page Error:**
image

image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Mozo, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦   🏃 

## Highlights 👍  
- Good job so far implementing state management using Redux!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- MIT license link in your ReadMe is broken, kindly endeavor to fix it.

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/pages/Categories.js (hunk 1 above)
Comment (felixDev22) [reply src/components/pages/Categories.js:17]: Thank you for your suggestions -  I have fixed the category link and the readme.
Screenshot 2023-01-26 at 12 11 38 PM

**Review by jerryowusu — APPROVED** [review]
Hi @felixDev22  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

CookedDelectableDolphin-size_restricted


## Highlights
All linters are passing :+1: 
Great effort in making the requested changes :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**